### ISSUE TITLE
How to set the parameters"lr"

### ISSUE THREAD

**Issue by htZhang25** [silversparro/wav2letter.pytorch#3]
Hi.Sorry to bother,I use my own dataset containing 240 vocabulary examples of GoogleSpeech,finding that after 500 epochs it still transcribes nothing,always ctc_blanks from epoch 2.
Could you tell me on which dataset you succeed in the validation.And how did you set "lr" and "learning-anneal"?1e-5 and 1.2 respectively？Thank you.

**Comment by vaibhav0195** [comment]
Hi , @htZhang25 
I experimented with kinda lots of values to get to these parameters. You can change the lr and  anneal by yourself.
Also try to look at the sample_rate param is that same that of your audios ?
Well I have trained and tested it on a proprietary dataset and have found satisfying results.

**Comment by htZhang25** [comment]
OK,Thanks.Have you experienced a nonnegligible period during which the transcription is blank.Or the transcription just begins from some random character but not blanks.I think these informations help me, maybe.

**Comment by vaibhav0195** [comment]
Hi @htZhang25 ,
I haven't faced the issue yet.
The thing i would like to point out here is that you only have 240 examples of training ? what is the duration of each example ? 
Whereas in the original paper they trained it on the data of 1000 hr to get the accuracy.
But since you have so less training examples you should try and reduce those convolution layers from the network and then try running the iterations.

**Comment by htZhang25** [comment]
Oh,I get it.I just used a shrunk dataset on purpose to see the val result quickly and the duration of each example is 1 second.I'll try a larger dataset like librispeech.Thanks.

**Comment by vaibhav0195** [comment]
Also try for audios between 5-7 seconds.
